src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,127 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Ticket {
+
+    private String event;
+    private Address address;
+    private String type;
+    private static double basicPrice;
+    private static double discount;
```

Comment:
Te pola nie mogą być statyczne, ponieważ to oznacza, że wszystkie bilety mają taką samą cenę i taką samą zniżkę